tests/wpt-harness/pre-harness.js

```diff
@@ -4,6 +4,9 @@ globalThis.GLOBAL = {
   isShadowRealm: function() { return false; },
 };
 
+function transferArrayBufferView () {}
+function ReadableByteStreamController () {}
```

Comment:
The real class for this is now exposed globally so we can remove this
```suggestion
function ReadableByteStreamController () {}
```